code/procs/jobprocs.dm

```diff
@@ -530,17 +530,29 @@ else if (istype(JOB, /datum/job/security/security_officer))\
 			src.stow_in_available(src.ears)
 		src.equip_if_possible(new /obj/item/device/radio/headset/deaf(src), SLOT_EARS)
 
-/// Equip items from body traits
-/mob/living/carbon/human/proc/equip_body_traits()
+/**
+Equip items from body traits.
+
+ * @param is_antagonist - If TRUE, the mob is an antagonist and will spawn with a pocket extended tank instead of a mini tank.
+
+**/
+/mob/living/carbon/human/proc/equip_body_traits(is_antagonist=FALSE)
```

Comment:
`is_antagonist` is a confusing name since it doesn't apply to most antagonists, why not just `extended_tank`?